app/common/public/locales/ko/translation.json

```diff
@@ -1,5 +1,5 @@
 {
-  "appiumInspector": "Appium Inspector",
+  "appiumInspector": "",
```

Comment:
I guess this removal was unexpected. Reverted in the crowdin so the next update will revert this removal.